index.js

```diff
@@ -0,0 +1,12 @@
+var displayValue = function(val){
```

Comment:
the JS is for stage2 and I would love to see you go back and fixup your stage 1 work. The main change being the calculator is a fixed size. It does not resize or change dimensions in anyway when the page or window resizes.